### ISSUE TITLE
Save chapter only if it's higher Option

### ISSUE THREAD

**Issue by Doomwhite** [Glagan/MyMangaDex#7]
The "Save chapter only if it's higher" option isn't really working here, whether or not the option is enabled or disabled it still updates my MAL with a chapter that is really higher than the previous one.

**Comment by Glagan** [comment]
Thanks, I forgot to add a condition, it's fixed now.
I uploaded the new version on the Firefox and Chrome store, it shouldn't take long to be available.

**Comment by Doomwhite** [comment]
Err, actually, the problem came back now.

**Comment by Doomwhite** [comment]
> Thanks, I forgot to add a condition, it's fixed now.
> I uploaded the new version on the Firefox and Chrome store, it shouldn't take long to be available.

Err, actually, the problem was fixed, but now it's back.

**Comment by Doomwhite** [comment]
The problem is happening with https://mangadex.org/title/19204/rikei-ga-koi-ni-ochita-no-de-shoumeishitemita
I'm on chapter 1, and whenever I open the chapter 16 up, it updates MAL.

**Comment by Glagan** [comment]
So you are currently on chapter 1 and when you open chapter 16 it updates MAL ? That's normal behaviour, the option doesn't do anything here. It's a bug if it's the other way and the option is disabled.

The options, if enabled, avoid updating MAL when you are reading a chapter *lower* than the current one.  
If it's disabled, it will just update to the current reading chapter regardless if it's higher or not than the current one.

But I think it's actually bugged, I just tried, weird, it worked when I pushed the fix... I will try to fix it for good.
